src/main/java/uk/gov/digital/ho/hocs/notify/application/RequestData.java

```diff
@@ -49,11 +66,6 @@ private String initialiseCorrelationId(HttpServletRequest request) {
         return !isNullOrEmpty(correlationId) ? correlationId : UUID.randomUUID().toString();
     }
 
-    private String initialiseUserId(HttpServletRequest request) {
```

Comment:
We don't want this to populate as 'anonymous' because we use it later as a lookup for an email address, we should keep it as null and we have that lookup in an `if null` check